### PR TITLE
[4.0] Fix searchtools js error

### PR DESCRIPTION
**Pull request by infograf768**
Pull Request for Issue https://github.com/joomla/joomla-cms/issues/29060

### Summary of Changes
taking care of case when there are no items.

### Testing Instructions
See https://github.com/joomla/joomla-cms/issues/29060

### Expected result after PR
No more error

@C-Lodder

### CODE REVIEW

File: build/media_source/system/js/searchtools.es6.js [hunk 1]
```diff
@@ -295,7 +295,9 @@ Joomla = window.Joomla || {};
       // If there are no active filters - remove the filtered caption area from the table
       if (activeFilterCount === 0) {
         const filteredByCaption = document.getElementById('filteredBy');
-        filteredByCaption.parentNode.removeChild(filteredByCaption);
+        if (filteredByCaption !== null) {
```
Comment (C-Lodder) [inline build/media_source/system/js/searchtools.es6.js]: You can just do `if (filteredByCaption) {`, but otherwise yes, this will fix the issue

Comment (infograf768) [reply build/media_source/system/js/searchtools.es6.js]: modified

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 67d89430900355fc10fed82100cd1a1c0ad5b2bd<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29066">issues.joomla.org/tracker/joomla-cms/29066</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 67d89430900355fc10fed82100cd1a1c0ad5b2bd<br /><br />Thanks<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29066">issues.joomla.org/tracker/joomla-cms/29066</a>.</sub>

**Comment by SharkyKZ** [comment]
RTC.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29066">issues.joomla.org/tracker/joomla-cms/29066</a>.</sub>

**Pull request merged**